test/SequenceEntity/data/sequenceRoot.js

```diff
@@ -0,0 +1,290 @@
+export const sequenceRoot = {
```

Comment:
This HM response is grabbed from the LMS copied pasted and removed most of the useless information. I kept information that isn't being used now since it will probably be required in the future.

Reply:
thanks, yeah that's all i was looking for.  Just a sanity check against real data

Reply:
It caught bugs!! YAY